tests/conftest.py

```diff
@@ -131,3 +134,11 @@ def _make_function(model: Model, version: t.Optional[str] = None, **kwargs) -> S
 @pytest.fixture
 def random_name() -> t.Callable:
     return lambda: f"generated_{random_id()}"
+
+
+def delete_cache(project_paths: str | t.List[str]) -> None:
```

Comment:
is there no method on cache to delet?

Reply:
There's currently none. And I'm not sure we need one outside tests

Reply:
What's the verdict? Good as is or this should be built into the cache code?

Reply:
@tobymao